pmm-app/tests/QAN/pages/qanDetailsFragment.js

```diff
@@ -28,6 +31,20 @@ module.exports = {
     compareCalculation(qpsvalue, result);
   },
 
+  checkExamplesTab() {
+    I.waitForVisible(this.getTabLocator('Example'), 30);
+    I.click(this.getTabLocator('Example'));
+    I.wait(5);
+    I.dontSeeElement(this.elements.noExamples);
+  },
+
+  checkExplainTab(){
+    I.click(this.getTabLocator('Explain'));
+    I.wait(5);
```

Comment:
No other way to handle this?

Reply:
I will think about it again. :)

Reply:
I was able to find pattern just for the Examples tab but not for Explain. So I deleted the wait in `checkExamplesTab()`